collections/guides/use/collect-cards-with-proxies.md

```diff
@@ -52,7 +52,7 @@ This new management Application will be used to create new Reactor Formulas, [Re
     - `reactor:create`
     - `proxy:create`
 
-[You can use this link to pre-fill the Create Application form in our Portal.](https://portal.basistheory.com/applications/create?name=Setup+Proxy&permissions=application%3Acreate&permissions=reactor%3Acreate&permissions=proxy%3Acreate&type=management)
+[You can use this link to pre-fill the Create Application form in our Portal.](https://portal.basistheory.com/applications/create?application_template_id=931b306f-6be7-405d-9a8d-1442a99dd2d7)
```

Comment:
Lets try to update this one to link with the exact list of management permissions